src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  let partner;
+
+  if (isMarried) {
+    if (sex === 'm') {
+      partner = `${partnerName} is my wife`;
+    } else {
+      partner = `${partnerName} is my husband`;
+    }
+  } else {
+    partner = `I am not married`;
+  }
```

Comment:
this code looks complicated, try to simplify it
you can create the separate variable for storing `wife` or `husband` depending on the sex value, and show needed message in return statement using created variables

Reply:
done